person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_accessor :id, :name, :age
+
+  def initialize(id, age, name = 'Unknown', parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    of_age? or @parent_permission
```

Comment:
- [ ]  To use the logical operator OR you should use `||` and not `or`  😄 

e.g

````
  def can_use_services? 
    is_of_age? || @parent_permission
  end
````